api/lib/schema.js

```diff
@@ -409,6 +413,12 @@ const typeDefs = gql`
     networkId: String!
   }
 
+  input NotificationSetting {
+    topic: String!
+    type: String!
+    remove: Boolean
```

Comment:
What is this remove thing?

Reply:
ah, it is commented out in line 60 in notificationsController.js. Ah I see, for the future to delete notifications registrations